ingestion-beam/src/main/java/com/mozilla/telemetry/Decoder.java

```diff
@@ -34,42 +37,46 @@ public static void main(String[] args) {
 
     final DecoderOptions.Parsed options = DecoderOptions.parseDecoderOptions(
         PipelineOptionsFactory.fromArgs(args).withValidation().as(DecoderOptions.class));
-
     final Pipeline pipeline = Pipeline.create(options);
-    final PTransform<PCollection<PubsubMessage>, ? extends POutput> errorOutput = options
-        .getErrorOutputType().write(options);
+    final List<PCollection<PubsubMessage>> errorCollections = new ArrayList<>();
```

Comment:
One of the bugs discovered here is that we were creating multiple transforms for each step's error output, but the configurations are identical, so they create potentially identical filenames and overwrite each other. We now collect error collections in a list and combine to one error output transform.